### PR TITLE
chore(pt-br): remove {{HTMLAttrDef}} macro

### PR DESCRIPTION
**Pull request by yin1999**
### Description

remove {{HTMLAttrDef}} macro

### Related issues and pull requests

Part of: https://github.com/orgs/mdn/discussions/263

### CODE REVIEW

File: files/pt-br/learn/html/howto/use_data_attributes/index.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: Utilizando data attributes
 slug: Learn/HTML/Howto/Use_data_attributes
 ---
 
-O [HTML5](/pt-BR/docs/Web/Guide/HTML/HTML5) foi criado pensando na extensibilidade dos dados que precisam ser associados a um determinado elemento mas não necessariamente tem um significado definido. [Atributos data-\*](/pt-BR/docs/Web/HTML/Global_attributes#attr-dataset) nos permite armazenar informações extras em elementos HTML padrões e semânticos, sem a necessidades de hacks como [classList](/pt-BR/docs/Web/API/Element.classList), atributos fora do padrão, propriedades extras no DOM ou o método depreciado [setUserData](/pt-BR/docs/Web/API/Node.setUserData).
+O [HTML5](/pt-BR/docs/Web/Guide/HTML/HTML5) foi criado pensando na extensibilidade dos dados que precisam ser associados a um determinado elemento mas não necessariamente tem um significado definido. [Atributos data-\*](/pt-BR/docs/Web/HTML/Global_attributes#dataset) nos permite armazenar informações extras em elementos HTML padrões e semânticos, sem a necessidades de hacks como [classList](/pt-BR/docs/Web/API/Element.classList), atributos fora do padrão, propriedades extras no DOM ou o método depreciado [setUserData](/pt-BR/docs/Web/API/Node.setUserData).
```
Comment (yin1999) [inline files/pt-br/learn/html/howto/use_data_attributes/index.md:6]: The "attr-" is not need after removing {{HTMLAttrDef}} macro

File: files/pt-br/web/html/global_attributes/index.md [hunk 2]
```diff
@@ -74,7 +74,7 @@ Além dos atributos globais HTML básicos, os seguintes atributos globais també
...
 - [`lang`](/pt-BR/docs/Web/HTML/Global_attributes/lang)
-  - : Participa da definição do idioma do elemento, o idioma no qual os elementos não-editáveis estão escritos, ou o idioma no qual elementos editáveis devem ser escritos. A Tag contém um único valor no formato definido no documento da IEFT [Tags para Identificação de Idiomas (BCP47)](http://www.ietf.org/rfc/bcp/bcp47.txt). [**xml:lang**](#attr-xml:lang) tem prioridade mais alta que [`lang`](/pt-BR/docs/Web/HTML/Global_attributes/lang).
+  - : Participa da definição do idioma do elemento, o idioma no qual os elementos não-editáveis estão escritos, ou o idioma no qual elementos editáveis devem ser escritos. A Tag contém um único valor no formato definido no documento da IEFT [Tags para Identificação de Idiomas (BCP47)](http://www.ietf.org/rfc/bcp/bcp47.txt). `xml:lang` tem prioridade mais alta que [`lang`](/pt-BR/docs/Web/HTML/Global_attributes/lang).
```
Comment (yin1999) [inline files/pt-br/web/html/global_attributes/index.md:77]: It has pointed to line 14, but as we has removed the HTMLAttrDef macro, this link would not work anymore, we have also removed the link in en-US.

**Review by josielrocha — APPROVED** [review]
LGTM ;)

**Comment by josielrocha** [comment]
Thank you for your help @yin1999!

**Pull request merged**